sps30-i2c/sps30.c

```diff
@@ -39,22 +39,28 @@
 #define SPS_CMD_START_MEASUREMENT_ARG 0x0300
 #define SPS_CMD_STOP_MEASUREMENT 0x0104
 #define SPS_CMD_READ_MEASUREMENT 0x0300
+#define SPS_CMD_START_STOP_DELAY_USEC 20000
 #define SPS_CMD_GET_DATA_READY 0x0202
 #define SPS_CMD_AUTOCLEAN_INTERVAL 0x8004
 #define SPS_CMD_GET_FIRMWARE_VERSION 0xd100
 #define SPS_CMD_GET_SERIAL 0xd033
 #define SPS_CMD_RESET 0xd304
+#define SPS_CMD_SLEEP 0x1001
+#define SPS_CMD_READ_DEVICE_STATUS_REG 0xd206
 #define SPS_CMD_START_MANUAL_FAN_CLEANING 0x5607
-#define SPS_CMD_DELAY_USEC 10000
-#define SPS_WRITE_DELAY_USEC 15000
+#define SPS_CMD_WAKE_UP 0x1103
+#define SPS_CMD_DELAY_USEC 5000
```

Comment:
Maybe the timing changes are the cause of the CI errors?

Reply:
Yup, when writing flash, the delay is 20ms.. I missed that when setting the auto clean interval - 15ms seemed to work, while 5ms was then too short.. 20ms is specified